hilti/toolchain/include/base/util.h

```diff
@@ -73,6 +73,13 @@ std::string fmt(const char* fmt, const Args&... args) {
 
 using hilti::rt::transform; // NOLINT(misc-unused-using-decls)
 
+/** Collect a view into a vector. */
+template<std::ranges::input_range R>
+auto toVector(R&& xs) {
+    // TODO(bbannier): replace callers with `std::ranges::to` in C++23 and remove this function.
```

Comment:
Could we make this like `TODO(C++23)` then when we upgrade (in 1 or 5 years) we just grep for that?

not sure if there will be more cases.